models/commons/src/client-logging/transformers.ts

```diff
@@ -12,6 +12,7 @@ const transformers = {
     buildFields: ['customer'],
   }),
   graphql: Transformer<TClientLogging, TClientLoggingGraphql>('graphql', {
+    buildFields: false,
```

Comment:
Since previously we needed to provide the `buildFields` array in order to specify which data models properties should be automatically built, omitting that property meant build no property at all.

Now that this field is not even needed, for those current cases where we don't actually want to build any property we need to explicitly return `false`.